src/lib.rs

```diff
@@ -3280,7 +3295,7 @@ fn map_darwin_target_from_rust_to_compiler_architecture(target: &str) -> Option<
     }
 }
 
-fn which(tool: &Path) -> Option<PathBuf> {
+fn which(tool: &Path, path_entries: Option<OsString>) -> Option<PathBuf> {
```

Comment:
Could this be split into a second function instead of taking an optional `path_entries` argument? It's non-obvious to me at least that given this signature what `path_entries` is and its behavior relative to `PATH`

Reply:
Would `override_path` be better? Or `override_PATH`? I see less point in duplicating this function...